metadata/pkg/drivers/aws/aws.go

```diff
@@ -69,28 +70,58 @@ func ObjectList(sess *session.Session, bucket *model.MetaBucket) error {
 		totSize += obj.Size
 		obj.StorageClass = *object.StorageClass
 
+		tags, err := svc.GetObjectTagging(&s3.GetObjectTaggingInput{Bucket: &bucket.Name, Key: &obj.ObjectName})
```

Comment:
please move to separate function, as original function body getting too long.

Reply:
done